frontend/src/App.js

```diff
@@ -34,7 +34,14 @@ const Routing = () => {
         <Route path="/register" element={<Signup />}></Route>
         <Route path="/" element={<Profile />}></Route>
         <Route path="/generatePass" element={<GeneratePass />}></Route>
-        <Route path="/viewPasses" element={<ViewPasses />}></Route>
+        <Route
+          path="/viewPasses"
+          element={<ViewPasses verify={false} abc={"1234"} />}
```

Comment:
Remove abcd

Reply:
Try to remove verify and observe the changes. And check how it impacts

Reply:
When I removed verify prop from viewPasses, the if-else statements in the useEffect only passed the else part, so every time viewUnverifiedPasses is called, viewPasses will never be called..